docs/guides/getting-started.md

```diff
@@ -0,0 +1,95 @@
+---
+page_title: "Getting Started"
+---
+
+# Getting Started
+
+## Installing Terraform
+
+See [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) for more information.
+
+## Declaring a Task
+
+On an empty directory:
+
+1. Create a directory named `directory` to save the task results.
+2. Create a file named `main.tf` with the following contents:
+
+```hcl
+terraform {
+  required_providers {
+    iterative = {
+      source  = "iterative/iterative"
+    }
+  }
+}
+
+provider "iterative" {}
+
+resource "iterative_task" "example" {
+  name  = "example"
+  cloud = "aws" # or any of: gcp, az, k8s
+
+  directory = "${path.root}/directory"
```

Comment:
Note that the `directory` will also be uploaded before the task starts. It's not just for `results` as that name would imply. Still, it may work for this use case and it's better than the `directory` _lapalissade_.